GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java

```diff
@@ -1707,13 +1758,107 @@ private void writeStatsAndGraphsSheet(
         }
     }
 
-    /**
-     * creates a cell in the row passed in and sets the style and value (if non-null)
-     */
+    private int writeCollectionStats(Map<QuestionGroupDto,
+            List<QuestionDto>> questionMap,
```

Comment:
Is the formatting maybe off? one param per line? not sure if flow has any code formatting conventions...

Reply:
I just like it that way.